inst/www/shared/selectize/accessibility/js/selectize-plugin-a11y.js

```diff
@@ -145,7 +145,14 @@ Selectize.define("selectize-plugin-a11y", function (options) {
     var original = self.destroy;
     return function () {
       self.accessibility.liveRegion.$region.remove();
-      return original.apply(this, arguments);
+      var destroyed = original.apply(this, arguments);
+      // Redefine plugin after destroying the instance so that selectizeInput()'s
+      // receiveMessage() can call destroy and update the instance without
+      // needing to reload this plugin
+      Selectize.define("selectize-plugin-a11y", selectize_accessibility_plugin);
```

Comment:
Is this line necessary? It looks like it's doing the same thing as 158.

Reply:
Yea, this line is the main point of the PR -- it makes sure the plugin is defined again after `destroy()` is called here 

https://github.com/rstudio/shiny/blob/5b12980b7a7012b0bff51892d536b8b4fc69fa6a/srcjs/input_binding_select.js#L56

and here

https://github.com/rstudio/shiny/blob/5b12980b7a7012b0bff51892d536b8b4fc69fa6a/srcjs/input_binding_select.js#L200

Reply:
What's strange is that calling `.destroy()` on an instance causes the global `Selectize` object to be altered (and then requires us to modify the `Selectize` object to alter it again).

Reply:
When I add a breakpoint and inspect the `Selectize.plugins` object right after `Selectize.define("selectize-plugin-a11y", ...)` is called, I see that `selectize-plugin-a11y` is listed:

![image](https://user-images.githubusercontent.com/86978/99581303-31198c80-29a6-11eb-8614-24440596c7f8.png)


However, if I wait until the JS console is idle and then look at it again, then the plugin does not show up. So there appears to be a bug where `selectize-plugin-a11y` is getting removed from `Selectize.plugins`.

![image](https://user-images.githubusercontent.com/86978/99581425-6aea9300-29a6-11eb-8750-daf2f674a173.png)

Reply:
I think the problem is that `selectize.min.js` is getting loaded twice. The first copy gets the accessibility plugin, and the second does not.

![image](https://user-images.githubusercontent.com/86978/99582527-d97c2080-29a7-11eb-9f3a-cdada4e87f67.png)

Reply:
With a breakpoint in `renderDependencies()` in `output_binding_html.js`, I can see that there's a `selectize-js: "0.12.4"` baked into the web page, but then there's also an dynamic "selectize", version: "0.12.0"`. So I think the issue is that there are two dependencies with different names, `selectize-js`, and `selectize`.


![image](https://user-images.githubusercontent.com/86978/99584092-1d702500-29aa-11eb-804c-7c1fb365c141.png)